std/socket.d

```diff
@@ -824,7 +794,7 @@ class InternetHost
     }
 }
 
-
+///
 @safe unittest
```

Comment:
We now verify with CircleCi that public examples are runnable (also part of the `style` Makefile target)

```
out/std_socket.d(13): Deprecation: std.socket.softUnittest is not visible from module std_socket
out/std_socket.d(13): Error: function std.socket.softUnittest is not accessible from module std_socket
```